Source/SelfService/Web/api/api.ts

```diff
@@ -108,17 +108,17 @@ export function getServerUrlPrefix(): string {
 
 export function getLatestRuntimeInfo(): LatestRuntimeInfo {
     return {
-        image: 'dolittle/runtime:7.8.1',
-        changelog: 'https://github.com/dolittle/Runtime/releases/tag/v7.8.1',
+        image: 'dolittle/runtime:8.0.0',
+        changelog: 'https://github.com/dolittle/Runtime/releases/tag/v8.0.0',
     };
 }
 
 
 export function getRuntimes(): LatestRuntimeInfo[] {
     return [
         {
-            image: 'dolittle/runtime:7.8.1',
-            changelog: 'https://github.com/dolittle/Runtime/releases/tag/v7.8.1',
+            image: 'dolittle/runtime:8.0.0',
+            changelog: 'https://github.com/dolittle/Runtime/releases/tag/v8.0.0',
```

Comment:
```suggestion
            getLatestRuntimeInfo(),
```

Just realized that we could just use the `getLatestRuntimeInfo()` method instead of having to keep this data up to date in 2 different places. Also my suggestion doesn't work as GitHub doesn't allow suggestions on deleted lines, but you get the idea. What do ya think?

Reply:
I think I get it. Could you verify the change I made?